doc/data/messages/c/confusing-consecutive-elif/good.py

```diff
@@ -0,0 +1,8 @@
+def myfunc(shall_continue: bool, shall_exit: bool):
+    if shall_continue:
+        if input("Are you sure?") == "y":
+            print("Moving on.")
+        else:
+            pass
+    elif shall_exit:
+        print("Exiting.")
```

Comment:
```suggestion
def user_confirmation():
    if input("Are you sure?") == "y":
        print("Moving on.")

def myfunc(shall_continue: bool, shall_exit: bool):
    if shall_continue:
        user_confirmation()
    elif shall_exit:
        print("Exiting.")
```
Another possibility imo is to create a function for the nested if/elif.

Reply:
Thanks, I will include both possibilities. 👍